WMIAdventure/frontend/src/js/components/battle/organisms/BattleView/BattleView.js

```diff
@@ -389,7 +399,17 @@ class BattleView extends React.Component {
     }
 
     compactCardActionDesktop = (userTurn) => {
-        console.log(userTurn);
+        setTimeout(() => {
+            if (userTurn) {
+                this.setState({
+                    compactCardOnTopScale: {user: '0', enemy: '1', middle: '1'},
+                });
+            } else {
+                this.setState({
+                    compactCardOnTopScale: {user: '1', enemy: '0', middle: '1'},
+                });
+            }
+        }, nextStepAnimationDuration);

```

Comment:
`this.setState({
    compactCardOnTopScale: {user: userTurn, enemy: !userTurn, middle: '1'}
`